src/_locales/de/messages.json

```diff
@@ -0,0 +1,254 @@
+{
+  "extensionName": {
+    "message": "Cookie AutoDelete",
+    "description": "Name of the extension."
+  },
+
+  "extensionDescription": {
+    "message": "Beherrsche deine Cookies! Lasse unbenutzte Cookies von deinen geschlossenen Tabs automatisch löschen.",
```

Comment:
I changed the text a little bit to say something like "Let unused cookies be automatically deleted" instead of "Auto Delete Unused Cookies". Sounds better in German in my opinion, though I can change this to the original text.